### PR TITLE
Update bash hook to test for .soroban-rpc-url

### PR DESCRIPTION
**Pull request by Kanaye**

### CODE REVIEW

File: _squirtle/bash-hook [hunk 1]
```diff
@@ -4,6 +4,10 @@ function _sq_set_secret() {
...
     fi
+    if [[ -f /workspace/.soroban-rpc-url ]]
+    then
+        export SOROBAN_RPC_URL=$(</workspace/.soroban-rpc-url)
+    fi
 }
 
-trap '_sq_set_secret' DEBUG
\ No newline at end of file
+trap '_sq_set_secret' DEBUG
```
Comment (kalepail) [inline _squirtle/bash-hook:13]: Do we need this line? What does it do?

**Review by kalepail — COMMENTED** [review]
Is this all we need? Seems like we'll need some more things in the _squirtle bot or the .gitpod.yml file.

**Comment by Kanaye** [comment]
That's all we need the hook already gets executed before each command in `CLI - Futurenet`, I just extended it to look for the second file and to read it into the environment if it exists.

File: _squirtle/bash-hook (hunk 1 above)
Comment (Kanaye) [reply _squirtle/bash-hook:13]: This line existed before, it sets our function as a "DEBUG trap" this is normally meant to debug bash scripts, as the trap is run before each executed command, but for our case it allows us to modify the env before running user commands.

**Pull request merged**